src/app/app.component.ts

```diff
@@ -1,13 +1,96 @@
-import { Component, ViewEncapsulation } from '@angular/core';
-
+import { Component } from '@angular/core';
+export class Hero {
+  id: number;
+  name: string;
+}
+const HEROES: Hero[] = [
+  { id: 11, name: 'Mr. Nice' },
+  { id: 12, name: 'Narco' },
+  { id: 13, name: 'Bombasto' },
+  { id: 14, name: 'Celeritas' },
+  { id: 15, name: 'Magneta' },
+  { id: 16, name: 'RubberMan' },
+  { id: 17, name: 'Dynama' },
+  { id: 18, name: 'Dr IQ' },
+  { id: 19, name: 'Magma' },
+  { id: 20, name: 'Tornado' }
+];
 @Component({
-  selector: 'toh-app',
-  encapsulation: ViewEncapsulation.None,
-  styles: [require('./app.component.scss')],
+  selector: 'my-app',
   template: `
-    <main>
-      <h1>Tour of Heroes</h1>
-    </main>
-  `
+    <h1>{{title}}</h1>
+    <h2>My Heroes</h2>
+    <ul class="heroes">
+      <li *ngFor="let hero of heroes"
```

Comment:
Whenever you use `ngFor`, you can boost performance by using the `trackBy` option. This will be something I call out when you're building your real site.